sootup.jimple.frontend/src/main/java/sootup/jimple/frontend/JimpleStringAnalysisInputLocation.java

```diff
@@ -66,7 +66,11 @@ public JimpleStringAnalysisInputLocation(
       JimpleConverter jimpleConverter = new JimpleConverter();
       classSource =
           jimpleConverter.run(
-              CharStreams.fromString(jimpleFileContents), this, path, bodyInterceptors);
+              CharStreams.fromString(jimpleFileContents),
+              this,
+              path,
+              bodyInterceptors,
+              new JimpleView(this));
```

Comment:
Why are we creating a new JimpleView here? We shouldn't do this in the InputLocations.